src/text-overflow/TextOverflow.d.ts

```diff
@@ -0,0 +1,12 @@
+import { PureComponent } from 'react';
+
+export type TextOverflowProps = {
+  label?: string | number,
+  popover?: object, // TODO(typo): Omit?
+  lazy?: boolean,
+  id?: string,
+  className?: string,
+  style?: object
```

Comment:
same comments as before: `TextOverflowProps` should reflect exactly the `@props` from `TextOverflow.js`.

- `label` is required and only string,
- `children` is missing
- `delayWhenLazy` is missing
- `popover` is not a prop of TextOverflow

Also, `style` should be `React.CSSProperties` because `object` represents any non-primitive type (which means also arrays, regexp, dates...)